substrate/frame/staking/src/benchmarking.rs

```diff
@@ -249,7 +249,7 @@ benchmarks! {
 		let original_bonded: BalanceOf<T>
 			= Ledger::<T>::get(&controller).map(|l| l.active).ok_or("ledger not created after")?;
 
-		let _ = asset::mint_existing::<T>(&stash, max_additional).unwrap();
+		let _ = asset::mint_existing::<T>(&stash, max_additional + asset::existential_deposit::<T>()).unwrap();
```

Comment:
bond_extra would not allow free part of balance to drop below ED after this change.